specs/author_spec.rb

```diff
@@ -0,0 +1,22 @@
+require '../classes/author'
+require '../classes/item'
+
+describe Author do
+  before :each do
+    @new_author = Author.new(first_name: 'Alex', last_name: 'Ramos')
+    @item = Item.new(genre: 'genre', author: 'author', label: 'label', publish_date: 20)
+  end
+
+  context 'Test a new author instance' do
+    it 'Should create a author when we call the class constructor' do
+      expect(@new_author).to be_instance_of Author
+    end
+  end
+
+  context 'Creation a item instance' do
+    it 'Add a new item to item array' do
+      @new_author.add_item(@item)
+      expect(@new_author.items.length).to eq 1
+    end
+  end
+end
```

Comment:
- [ ] The `author` and `label` specs crashes when I tried running them but other tests passed. Let's fix the `author` and `label` specs so that all your test passes 💯

Reply:
Hi, @julie-ify  can you kindly tell us what type of error is giving to you? because in our local environment both specs `author` and `label` work as expected.

Reply:
@AlexRS90 here 👇 

<img width="370" alt="Loom_E7XrVAZWvj" src="https://user-images.githubusercontent.com/81468675/153527564-b09087f0-7d15-4955-b9c6-b9b8007102a3.png">

Reply:
Thank you very much @julie-ify, we'll work on them! 🤓